conf/base_config.yaml

```diff
@@ -31,7 +31,7 @@ simulator:
         field_list:  # list of dictionaries, each with run, camcol, and list of fields
           - run: 94
             camcol: 1
-            fields: ${range:12,500,50}
+            fields: [12]
```

Comment:
@aakashdp6548 Is this change to `fields` in `base_config.yaml` OK with you? (Will this affect your code?)

Reply:
Yup, should be fine! The default in base_config should actually be [12] anyway since the other files may not be downloaded, I made the change in #775 as well